firefox-ios/firefox-ios-tests/Tests/XCUITests/TodayWidgetTests.swift

```diff
@@ -11,10 +11,10 @@ let editHomeScreenButton = "com.apple.springboardhome.application-shortcut-item.
 let removeWidgetButton = "com.apple.springboardhome.application-shortcut-item.remove-widget"
 
 // Widget Buttons Identifier
-let goToCopiedLink = springboard.buttons["Go to Copied Link"]
-let newPrivateSearch = springboard.buttons["New Private Search"]
-let newSearch = springboard.buttons["New Search"]
-let clearPrivateTabs = springboard.buttons["Clear Private Tabs"]
+var goToCopiedLink = springboard.buttons["Go to Copied Link"]
+var newPrivateSearch = springboard.buttons["New Private Search"]
+var newSearch = springboard.buttons["New Search"]
+var clearPrivateTabs = springboard.buttons["Clear Private Tabs"]
```

Comment:
did you need to change that because of lint? if not, since their value do not change, it may be better to leave let

Reply:
No, I changed because I need to change the value of the variables for iOS 16. For iOS 16 I used static text instead of a button

Reply:
<img width="698" alt="Screenshot 2024-10-30 at 11 24 28" src="https://github.com/user-attachments/assets/23a11a3b-9d3a-46b7-b511-960e3a5f8343">

Reply:
oh okay, sorry I missed that